### PR TITLE
Statifying services and layersConfig

### PR DESCRIPTION
**Pull request by procrastinatio**
to avoid using apache proxy (for offline uses, files have to be on the same server)
Files are written to `prd/cache`

The `dev`mode still uses the remote service

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -307,6 +309,11 @@ prd/geoadmin.appcache: src/geoadmin.mako.appcache \
 	    --var "api_url=$(API_URL)" \
 	    --var "public_url=$(PUBLIC_URL)" $< > $@
 
+prd/cache/:
+	mkdir -p $@
+	curl -q -o prd/cache/services http:$(API_URL)/rest/services
+	$(foreach lang, $(LANGS), curl -q -o prd/cache/layersConfig.$(lang) http:$(API_URL)/rest/services/all/MapServer/layersConfig?lang=$(lang);)
+
```
Comment (gjn) [inline Makefile]: I think we need a clean cache command as well, to remove old files. Or by default, remove the created files when make clean is called.

**Comment by gjn** [comment]
Don't we risk to hit the same geoip/spamhouse problem when API_URL is api3.geo.admin.ch? (As the request is done from within amazon network)?

File: Makefile (hunk 1 above)
Comment (procrastinatio) [inline Makefile]: `make clean` removes directory `prd`

**Comment by procrastinatio** [comment]
Don't think so. Curling from dev or prod to api3.geo.admin.ch is OK, while proxying is not.

File: apache/app.mako-dot-conf [hunk 2]
```diff
@@ -37,8 +37,13 @@ RedirectMatch ^${apache_base_path}$ ${apache_base_path}/
 
 Alias ${apache_base_path}/src ${apache_base_directory}/src
 Alias ${apache_base_path}/ ${apache_base_directory}/prd/
-RewriteRule ${apache_base_path}/(src|[0-9]+)/layersConfig http:${api_url}/rest/services/all/MapServer/layersConfig [P]
-RewriteRule ${apache_base_path}/(src|[0-9]+)/services http:${api_url}/rest/services [P]
+RewriteRule ${apache_base_path}/src/layersConfig http:${api_url}/rest/services/all/MapServer/layersConfig [P]
```
Comment (oterral) [inline apache/app.mako-dot-conf]: `https`?

**Comment by oterral** [comment]
PR rebased

**Comment by procrastinatio** [comment]
Parsing the downloaded files with python and displaying the topics and layers number as a check. Should be enough for now.

**Comment by gjn** [comment]
Deployed to int. Works well.

**Pull request merged**